.github/scripts/summarize_tests.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
```

Comment:
Typically, you should be using general set of options for bash scrip like 

```
set -eEuo pipefail
```
It will protect script execution and exit if there is an issue rather then continue bringing misleading results

Reply:
Thanks - amended!